packages/bot-web-ui/src/stores/server-side-bot-store.ts

```diff
@@ -378,8 +378,20 @@ export default class ServerBotStore {
         try {
             const items = [[localize('Journal')]];
             [...this.journal].map(item => {
-                const arr = [item.msg];
-                items.push(arr);
+                let combinedMessage;
```

Comment:
@mayuran-deriv we use snake case for variables and camel case for functions in the Bot project
```suggestion
                let combined_message;
```